docs/guides/dtensor-tp-accuracy.md

```
@@ -0,0 +1,242 @@
+
+Combining the three images for observation, it is not necessarily true that abnormal `token_mult_prob_error` leads to abnormal reward and validation accuracy. This occurs for several reasons:
+
+1. **Spike pattern instead of continuous growth**: In many runs, `token_mult_prob_error` shows frequent spikes rather than a monotonically increasing trend, indicating that training is unstable but not fundamentally broken.
+2. **Stochastic occurrence of spikes**: The abnormal `token_mult_prob_error` is itself unstable; even with the same batch of data, spikes may not appear in every run.
+3. **Dilution effect with large datasets**: When the dataset is sufficiently large and no critical samples are repeatedly affected, these extreme but sporadic spikes may have limited impact on aggregate metrics, so the final reward and validation accuracy may not exhibit significant deviations.
+
+<img src="../assets/dtensor-tp-accuracy/image-20260111142255534.png" alt="image-20260111142255534" style="zoom: 33%;" />
+
+<p align="center"><em>Fig 2: The reward of Qwen3-4B</em></p>
+
+<img src="../assets/dtensor-tp-accuracy/validation_accuracy.png" style="zoom:33%;" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)